src/mobu/services/business/base.py

```diff
@@ -114,21 +111,21 @@ def __init__(
 
     # Methods that should be overridden by child classes if needed.
 
-    async def startup(self) -> None:
+    async def startup(self) -> None:  # noqa: B027
```

Comment:
What about `return None` for these instead of disabling linting? That is, if that gets rid of the warning which I don't actually know if it does...

Reply:
Oh, good call, I will try that and see if that suppresses the Ruff warning.